package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "js-todolist",
+  "version": "1.0.0",
+  "description": "4th Microverse JS Project",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
It's a good practice to add more scripts to your package file. You can create at least 2 scripts, one for productions(build) and another one to the development stage(dev). It can be something like:
```
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
```